src/lib/api/infrastructure.ts

```diff
@@ -27,8 +26,7 @@ export const mnestixFetch = ():
     | undefined => {
     return {
         fetch: async (url: RequestInfo, init?: RequestInit) => {
-            const text = await performServerFetch(url, await initializeRequestOptions(await getBearerToken(), init));
-            return new Response(text);
+            return await fetch(url, await initializeRequestOptions(await getBearerToken(), init));
```

Comment:
I know why you did this change, but we need to make sure to revert this once we have a production-ready solution for it

Reply:
I don't know why you did this change, but I'm trusting Melin on this :D.